### PR TITLE
Updated camera stacking docs. [Skip CI]

### PR DESCRIPTION
**Pull request by phi-lira**
Update Camera Stacking docs.

### CODE REVIEW

File: com.unity.render-pipelines.universal/Documentation~/universal-additional-camera-data.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # The UniversalAdditionalCameraData component
 
-[UniversalAdditionalCameraData](../api/UnityEngine.Rendering.Universal.UniversalAdditionalCameraData.htmll) is a component that the Universal Render Pipeline (URP) uses for internal data storage. The UniversalAdditionalCameraData component allows URP to extend and override the functionality of Unity's standard Camera component.
+The [Universal Additional CameraData](../api/UnityEngine.Rendering.Universal.UniversalAdditionalCameraData.htmll) component is a component that the Universal Render Pipeline (URP) uses for internal data storage. The UniversalAdditionalCameraData component allows URP to extend and override the functionality and appearance of Unity's standard Camera component.
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/universal-additional-camera-data.md]: seems like a typo that has extra `l` in `html`

File: com.unity.render-pipelines.universal/Documentation~/cameras-advanced.md [hunk 2]
```diff
@@ -27,22 +27,24 @@ At the start of its render loop, an [Overlay Camera](camera-types-and-render-typ
...
-3. Unity sorts the Base Cameras that render to Render Textures into priority order, so that Cameras with a higher priority value are drawn last.
+3. Unity sorts the Base Cameras that render to Render Textures into **Priority** order, so that Cameras with a higher **Priority** value are drawn last.
 4. For each Base Camera that renders to a Render Texture, Unity performs the following steps:
     1. Cull the Base Camera
     2. Render the Base Camera to the Render Texture
     3. For each [Overlay Camera](camera-types-and-render-type.md#overlay-camera) that is part of the Base Camera's [Camera Stack](camera-stacking.md), in the order defined in the Camera Stack:
         1. Cull the Overlay Camera
         2. Render the Overlay Camera to the Render Texture
-5. Unity sorts the Base Cameras that render to the screen into priority order, so that Cameras with a higher priority value are drawn last.
+5. Unity sorts the Base Cameras that render to the screen into **Priority** order, so that Cameras with a higher **Priority** value are drawn last.
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/cameras-advanced.md]: to the screen is redundant and incorrect (base camera can render to a render texture).

File: com.unity.render-pipelines.universal/Documentation~/cameras.md [hunk 3]
```diff
@@ -2,13 +2,12 @@
...
 
-The most notable ways in which the Camera system in URP differs from the standard Unity Camera are:
-
-* The [UniversalAdditionalCameraData](universal-additional-camera-data.md) component, which allows URP to store additional Camera-related data
+* The [Universal Additional Camera Data](universal-additional-camera-data.md) component, which extends the Camera component's functionality and allows URP to store additional Camera-related data
 * The [Render Type](camera-types-and-render-type.md) setting, which defines the two types of Camera in URP: Base and Overlay
 * The [Camera Stacking](camera-stacking.md) system, which allows you to layer the output of multiple Cameras into a single combined output
 * The [Volume](Volumes.md) system, which allows you to apply [post-processing effects](integration-with-post-processing.md) to a Camera based on a given Transform's position within your Scene
+* The [Camera component](camera-component-reference.md), which exposes URP-specific properties in the Inspector
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/cameras.md]: maybe should be first link?

File: com.unity.render-pipelines.universal/Documentation~/cameras-advanced.md (hunk 2 above)
Comment (reallyfancy) [reply com.unity.render-pipelines.universal/Documentation~/cameras-advanced.md]: See step 4, above - that relates to Render Textures. Step 5 relates to the screen.

Is this correct?

File: com.unity.render-pipelines.universal/Documentation~/cameras.md (hunk 3 above)
Comment (reallyfancy) [reply com.unity.render-pipelines.universal/Documentation~/cameras.md]: If that page were more than just a component reference, I'd agree - but in this case I think the order is correct. That page won't make much sense to the user unless they've read everything else in the section.

Comment (phi-lira) [reply com.unity.render-pipelines.universal/Documentation~/cameras.md]: LGTM.

**Pull request merged**